packages/@dcl/sdk/package.json

```diff
@@ -18,11 +18,14 @@
   "dependencies": {
     "@dcl/dcl-rollup": "file:../dcl-rollup",
     "@dcl/ecs": "file:../ecs",
-    "@dcl/ecs-math": "^2.0.1-20221108141807.commit-a1344cb",
+    "@dcl/ecs-math": "^2.0.1-20221129185242.commit-40495c1",
     "@dcl/js-runtime": "file:../js-runtime",
     "@dcl/kernel": "^2.0.0-3568680164.commit-ec0d714",
     "@dcl/react-ecs": "file:../react-ecs",
     "@dcl/unity-renderer": "^1.0.64444-20221125155434.commit-fcef3c4"
   },
-  "minCliVersion": "3.12.3"
+  "minCliVersion": "3.12.3",
+  "files": [
+    "dist"
+  ]
```

Comment:
This won't work, the @dcl/sdk package exports the .d.ts right in the root folder to enable

`import {Vector3} from '@dcl/ecs/math'` instead of `import {Vector3} from '@dcl/ecs/dist/math'`

Reply:
what if we copy the package.json inside the dist folder, and publish with cwd: '/dist/' ?
Compiling all the `.js` files at the root level is kind of annoying for me.

Reply:
we can do this after release, but would love to have an alternative solution 😄